implementations/validator/validator.py

```diff
@@ -56,15 +62,14 @@ def make_html(self, results):
 
     def format_response(self, data, fmt):
         if fmt == "html":
-            response.content_type="text/html"
+            response.content_type = "text/html"
             return self.make_html(data)
         else:
-            response.content_type="application/json"
+            response.content_type = "application/json"
             return json.dumps(data)
 
     def do_test(self):
         url = request.query.get('url', '')
-        typ = request.query.get('type', 'manifest')
```

Comment:
That's what I was thinking – since it wasn't being used I figured it was better not to have it floating around. Plus this way I didn't have to add a flake8 exception for that line.